src/sphtfunc.jl

```diff
@@ -156,15 +168,23 @@ function map2alm(map::PolarizedMap{Float64, RingOrder};
 end
 
 # convert maps to Float64
-function map2alm(map::Map{T, RingOrder}; lmax=nothing, mmax=nothing, 
-        niter::Integer=3) where T <: Real
+function map2alm(
+    map::Map{T, RingOrder, Array{T, 1}};
```

Comment:
This particular wrapper function can dispatch on `{T, AA <: AbstractArray{T, 1}}` since it calls `convert` on the array type into `Array` internally. Most custom array types define `Base.convert` to `Array{T,N}`. When we write some docs for the SHT, it may be helpful to note that custom array types will have a Array{Float64,1} copy made of them since we are dealing with C code which must receive that form.

Reply:
I missed that, now it's been fixed.